oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -370,7 +372,44 @@ public boolean isRequired(HttpResponse response) {
   }
 
   /**
-   * Returns whether the scopes are empty, meaning createScoped must be called before use.
+   * Returns a Google Id Token from the metadata server on ComputeEngine.
+   *
+   * @param targetAudience The aud: field the IdToken should include.
+   * @param options        List of Credential specific options for for the
+   *                       token. Currently unused for ServiceAccountCredentials.
+   * @throws IOException   if the attempt to get an IdToken failed
+   * @return IdToken object which includes the raw id_token, expiration and
```

Comment:
Can we add a `@see` link for the docs for this API

Reply:
we really don't have any documentation that covers this flow..I am working on publishing them (internal ref b/136495350)